repo/packages/W/weavescope/2/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "packagingVersion": "3.0",
+  "name": "weavescope",
+  "version": "1.2.1",
+  "tags": ["microservices", "visualisation", "monitoring"],
+  "maintainer": "help@weave.works",
+  "description": "Live visualisation, monitoring and management\n\nWeave Scope produces a live interactive map of your containerized services and applications with no coding and no instrumentation",
```

Comment:
@ryadav88 is there a chance we could merge this to bring the package up-to-date for existing users and we shall take the action to follow-up with documentation PR in a few weeks, what do you think?